.openpublishing.publish.config.json

```diff
@@ -263,6 +263,10 @@
       "ReferenceTOC": "docs/docs-ref-autogen/powerpoint_1_4/toc.yml",
       "ConceptualTOCUrl": "/office/dev/add-ins/toc.json"
     },
+    {
+      "ReferenceTOC": "docs/docs-ref-autogen/powerpoint_1_5/toc.yml",
```

Comment:
I'm not seeing powerpoint-js-1.5 in the moniker_ranges array

Reply:
Thanks for the review! I fixed both issues you caught.